hack/local-up-cluster.sh

```diff
@@ -1146,15 +1146,18 @@ function tolerate_cgroups_v2 {
 }
 
 function install_cni {
-  cni_plugin_sha=CNI_PLUGINS_${CNI_TARGETARCH^^}_SHA256SUM
```

Comment:
This is backward incompatible I think. It practically means ignoring CNI_TARGETARCH environment variable.

Reply:
The `CNI_TARGETARCH` variable is expected to be deleted and replaced by the `host_arch`

Reply:
Yes, and this would break developers workflows if they set `CNI_TARGETARCH`.
Even if we consider doing this we should at least mention it in release notes.

Reply:
Given the addition of the ability to automatically recognize the Arch of the CNI binary, I would prefer to remove `CNI_TARGETARCH` and add release note.

Reply:
I'd suggest not to avoid breaking backwards compatibility. In this case it means only do automatic recognition when `CNI_TARGETARCH` is not set.

Reply:
All right